pkg/server/server_test.go

```diff
@@ -0,0 +1,11 @@
+package server
+
+import "testing"
+
+func TestVerify(t *testing.T) {
```

Comment:
I think what we want to do for testing is replace this with an integration test that says:
1. If the auth middleware fails verification, I get a 401 and no data
2. If the auth middleware passes verification, I get a 200 with my data

Reply:
We'll also want to look into running a full e2e test in staging. This gets hard with how EUA is setup though.

Reply:
We should add a story to do this once e2e tests are up and running

Reply:
Let's delete this test for now? Before having anything running on staging, this should get tested in some fashion. If that's part of this upcoming search epic, let's do it.